etc/manifest.json

```diff
@@ -2,6 +2,7 @@
   "product_name": "YODAOS-Device",
   "version": "1.0.0",
   "capabilities": {
+    "main-visibility": "voice",
```

Comment:
How about using an array to represent this just like:

```js
"visibility": [ "voice", "graphics" ]
```

Reply:
There is nothing to do with non-main visibilities AFAIK.

Reply:
Is there possible to own a priority for these visibilities in the future?

Reply:
Yet `main-visibility` still be the unique one.